libs/cardano-ledger-core/src/Cardano/Ledger/Address.hs

```diff
@@ -364,14 +366,10 @@ getByron =
 -- to help enforce that people do not post huge ones on the chain.
 bootstrapAddressAttrsSize :: BootstrapAddress c -> Int
 bootstrapAddressAttrsSize (BootstrapAddress addr) =
-  -- I'm sorry this code is formatted so weridly below.
-  -- It is to apease the capricious god Ormolu. A sacrifice is demanded!
-  maybe
-    0
-    (BS.length . Byron.getHDAddressPayload)
-    (Byron.aaVKDerivationPath (Byron.attrData attrs))
-    + Byron.unknownAttributesLength attrs
+  maybe 0 payloadLen derivationPath + Byron.unknownAttributesLength attrs
```

Comment:
hehe nice solution to the formatting problem :laughing:

Reply:
Yeah, I got tired of reading the comment about ormolu gods